src/app/features/public-form/edit-public-form-field/edit-public-form-field.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from "@angular/core";
+import { EditComponent } from "app/core/entity/default-datatype/edit-component";
+import { DynamicComponent } from "app/core/config/dynamic-components/dynamic-component.decorator";
+
+import { Entity, EntityConstructor } from "app/core/entity/model/entity";
+import { EntityRegistry } from "app/core/entity/database-entity.decorator";
+import { AdminEntityFormComponent } from "app/core/admin/admin-entity-details/admin-entity-form/admin-entity-form.component";
+@Component({
+  selector: "app-edit-public-form-field",
+  standalone: true,
+  imports: [AdminEntityFormComponent],
+  templateUrl: "./edit-public-form-field.component.html",
+})
+@DynamicComponent("EditPublicFormField")
+export class EditPublicFormFieldComponent<T extends Entity = Entity>
+  extends EditComponent<T>
+  implements OnInit
+{
+  entityConstructor: EntityConstructor;
+  publicConfigDetailsView: any;
+
+  constructor(private entities: EntityRegistry) {
+    super();
+  }
```

Comment:
The new angular "inject()" function helps to do DI without contructor. That way we can skip that annoying call to super() constructor.

```suggestion
  private entities = inject(EntityRegistry);
```
(will need an additional import, check Quick Fix suggestions)